src/index.scss

```diff
@@ -142,3 +142,39 @@ button.btn.btn-command-block--right90:hover,
 button.btn.btn-command-block--right180:hover {
     background-color: $c2lc-right-command-hover-color;
 }
+
+
+// need to make play button rule complex to override React bootstrap button
+button.PlayControlButton:not(.PlayControlButton--disabled),
```

Comment:
Is there anything that has the `PlayControlButton` class that isn't a button?  I know we got in the habit of being specific to beat react-bootstrap, but we seem to be able to do without the `button` part for the `RefreshButton` styles (and the now-removed `StopButton` styles).

Reply:
This was to compete against .IconButton rules.